### PR TITLE
Slimes and Booze fixes

### LINKED ISSUE #5376: Drunkess is broken
Drunk mechanics are broken right now, you don't suffer any effect from it if you drink alcohol.

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Water was killing slimes super slowly because of an oversight. This fixes that oversight by making it so that the more water you spray on a slime, the faster you kill it. This applies to all slime killing chemicals.

Alcohol had little effect due to an overnerf by Skull. This fixes it so that drinking an entire bottle of whiskey is now like drinking an entire bottle of whiskey. Metabolism values for alcohol were also tweaked to process faster.

Fixes #5376

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm [hunk 1]
```diff
@@ -235,10 +236,12 @@
...
-/datum/reagent/water/affect_touch(var/mob/living/carbon/M, var/alien, var/removed)
-	if(istype(M, /mob/living/carbon/slime))
-		var/mob/living/carbon/slime/S = M
-		S.adjustToxLoss(8 * removed) // Babies have 150 health, adults have 200; So, 10 units and 13.5
+/datum/reagent/water/affect_touch(var/mob/living/carbon/slime/S, var/alien, var/removed)
+	if(istype(S))
+		//Adult slimes have 200 HP, baby slimes have 100 HP.
+		//45u of water should kill a baby slime in 10 seconds.
+		//Wiki: Deals volume * 0.44 damage to slimes per second.
+		S.adjustToxLoss((volume/45) * (removed/REM) * 100 * (1/10))
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Core.dm]: That can be simplified a bit.
`
*100*(1/10) -> *10
`

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 2]
```diff
@@ -354,6 +351,20 @@
 			return
 	M.clean_blood()
 
+
+	if(istype(M,/mob/living/carbon/slime))
+		var/mob/living/carbon/slime/S = M
+		//Adult slimes have 200 HP, baby slimes have 100 HP.
+		//20u of cleaner should kill a baby slime in 10 seconds.
+		//Wiki: Deals volume damage to slimes per second.
+		S.adjustToxLoss( (volume/20) * (removed/REM) * 100 * (1/10) )
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: Same as above

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm [hunk 3]
```diff
@@ -354,6 +351,20 @@
...
+		if(!S.client)
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: I am not sure if those changes wouldnt make it a bit too easy to deal with slime outbreaks.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm [hunk 4]
```diff
@@ -301,12 +301,16 @@
...
 /datum/reagent/toxin/fertilizer/monoammoniumphosphate/affect_touch(var/mob/living/carbon/slime/S, var/alien, var/removed)
 	if(istype(S))
-		S.adjustToxLoss(8 * removed)
-		if(!S.client && S.Target)
-			S.Target = null
-			++S.Discipline
+		//Adult slimes have 200 HP, baby slimes have 100 HP.
+		//30u of extinguisher should kill a baby slime in 10 seconds.
+		//Wiki: Deals volume * 0.66 damage to slimes per second.
+		S.adjustToxLoss( (volume/30) * (removed/REM) * 100 * (1/10) )
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm]: Here aswell

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm (hunk 3 above)
Comment (BurgerLUA) [reply code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Other.dm]: Reduced

**Pull request merged**